### ISSUE TITLE
Fresh install with Homebrew & npm gives unbound variable error: 

### ISSUE THREAD

**Issue by daniel-beard** [jez/git-heatmap#1]
- Installed bars with this: `npm install -g https://github.com/jez/bars.git`
- Installed git-heatmap with this: `brew install jez/formulae/git-heatmap`
- Running `git heatmap` in a git repository gives me this:

```
$ git heatmap
/usr/local/bin/git-heatmap: line 57: argv[@]: unbound variable
No data. Provide data on stdin.

Ascii bar charting

Usage:
  bars [--width <n>] [--bar <char>] [--sort]

Input format:

  <count> <key>

  Provide data on stdin.
  Leading, trailing, or extra spaces in the middle are ignored.

Options:
  --width <n>   Restrict chart to <n> characters. [default: 60]
  --bar <char>  Use <char> to draw the bars. [default: #]
  --sort        Sort the list in descending order

Example:
  bars --width 40 --bar █ --sort < my-data.txt
```

**Comment by jez** [comment]
What version of bash are you using?

**Comment by daniel-beard** [comment]
@jez the default macOS high sierra version: 
```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
```

**Comment by jez** [comment]
Looks like I'm using some bash 4.0 specific features.

Try:

- `brew install bash`
- Make sure `/usr/local/bin/bash` comes before `/bin/bash`
- Try `git heatmap` again

**Comment by GoodVibesDevelopment** [comment]
I had the same problem on Ubuntu. My bash version: 
`GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)` 

This works for me, in line 57. in git-heatmap file:
`git log --name-status --pretty=format: -- "${argv[@]}" | cut -f 2-`
just replace `${argv[@]}` by `$@`:
`git log --name-status --pretty=format: -- "$@" | cut -f 2-`.

**Comment by kewlar** [comment]
> just replace `${argv[@]}` by `$@`

Fixed the same issue for me, on ZSH (`zsh 5.2 (x86_64-apple-darwin16.0)`) on macOS Sierra.

Thanks, @GoodVibesDevelopment !

**Comment by jez** [comment]
For what it’s worth, those aren’t the same changes. The argv solution
filters out all the flags that aren’t directly relevant to git log. For
example, if you passed the —width or —base flag, it would fail with that
diff.

**Comment by jez** [comment]
I'm pretty sure I've fixed this in 0.10.2.

```
brew upgrade git-heatmap
```

**Comment by Danl2620** [comment]
I'm getting the same result on macOS even though I've upgraded bash to 5.0 and I'm on version 0.10.3 of git-heatmap.  Is there anything else I can try?
